apps/cli/src/node/default.env

```diff
@@ -1,76 +1,26 @@
+S6_VERBOSITY="2"
 S6_CMD_WAIT_FOR_SERVICES_MAXTIME=${SM_DEADLINE_MACHINE:-30000}
-# global
-RUST_LOG="${RUST_LOG:-info}"
-## shared
-### redis
-REDIS_ENDPOINT=${REDIS_ENDPOINT:-redis://127.0.0.1:6379}
-### contract-address-file
-DAPP_CONTRACT_ADDRESS_FILE=/usr/share/sunodo/dapp.json
-### session-id
-SESSION_ID=default_session_id
-### chain-id
-CHAIN_ID=31337
-TX_CHAIN_ID=31337
-### server-manager-endpoint
-SERVER_MANAGER_ENDPOINT=http://localhost:5001
-SERVER_MANAGER_ADDRESS=localhost:5001
-SERVER_MANAGER_LOG_LEVEL="${SERVER_MANAGER_LOG_LEVEL:-info}"
-REMOTE_CARTESI_MACHINE_LOG_LEVEL="${REMOTE_CARTESI_MACHINE_LOG_LEVEL:-info}"
-### state-server-endpoint
-SC_GRPC_ENDPOINT=http://localhost:50051
 
-### database
-POSTGRES_ENDPOINT="${POSTGRES_ENDPOINT:-postgres://postgres:password@database:5432/postgres}"
-
-# dispatcher
-## uses redis
-## uses chain-id
-## uses state-server-endpoint
-DAPP_DEPLOYMENT_FILE=/usr/share/sunodo/dapp.json
-ROLLUPS_DEPLOYMENT_FILE=/usr/share/sunodo/localhost.json
-RD_EPOCH_DURATION="${RD_EPOCH_DURATION:-86400}"
-SC_DEFAULT_CONFIRMATIONS=1
-
-# authority-claimer
-## uses redis
-## uses state-server-endpoint
-## uses chain-id (TX_CHAIN_ID acctually)
-TX_CHAIN_IS_LEGACY: "false"
-TX_DEFAULT_CONFIRMATIONS: "2"
-TX_SIGNING_MNEMONIC="test test test test test test test test test test test junk"
-TX_PROVIDER_HTTP_ENDPOINT=http://anvil:8545
-
-# state-server
-SF_GENESIS_BLOCK=1
-SF_SAFETY_MARGIN=1
-BH_HTTP_ENDPOINT=http://anvil:8545
-BH_WS_ENDPOINT=ws://anvil:8545
-BH_BLOCK_TIMEOUT="${BLOCK_TIMEOUT:-8}"
-
-# advance-runner
-## uses redis
-## uses contract-address-file
-## uses session-id
-## uses chain-id
-## uses server-manager-endpoint
-PROVIDER_HTTP_ENDPOINT=http://anvil:8545
-SNAPSHOT_DIR="/var/opt/cartesi/machine-snapshots"
-SNAPSHOT_LATEST="/var/opt/cartesi/machine-snapshots/latest"
-
-# inspect-server
-## uses session-id
-## uses server-manager-endpoint (acctually uses SERVER_MANAGER_ADDRESS)
-INSPECT_SERVER_ADDRESS=0.0.0.0:5005
-
-# indexer
-## uses redis
-## uses contract-address-file
-## uses chain-id
-
-# graphql-server
-## uses postgres-config
-GRAPHQL_HOST=0.0.0.0
-GRAPHQL_PORT=4000
\ No newline at end of file
+# cartesi/rollups-node
+CARTESI_LOG_LEVEL="info"
+CARTESI_LOG_TIMESTAMP="true"
+CARTESI_HTTP_PORT="10000"
+CARTESI_FEATURE_HOST_MODE="false"
+CARTESI_FEATURE_READER_MODE="false"
+CARTESI_EPOCH_DURATION="120"
+CARTESI_BLOCKCHAIN_ID="31337"
+CARTESI_BLOCKCHAIN_HTTP_ENDPOINT="http://anvil:8545"
+CARTESI_BLOCKCHAIN_WS_ENDPOINT="ws://anvil:8545"
+CARTESI_BLOCKCHAIN_IS_LEGACY="false"
+CARTESI_BLOCKCHAIN_GENESIS_BLOCK="1"
+CARTESI_BLOCKCHAIN_FINALITY_OFFSET="1"
+CARTESI_CONTRACTS_APPLICATION_ADDRESS="0x70ac08179605AF2D9e75782b8DEcDD3c22aA4D0C"
+CARTESI_CONTRACTS_APPLICATION_DEPLOYMENT_BLOCK_NUMBER="1"
+CARTESI_CONTRACTS_INPUT_BOX_DEPLOYMENT_BLOCK_NUMBER="1"
+CARTESI_CONTRACTS_HISTORY_ADDRESS="0x4FF8BD9122b7D91d56Dd5c88FE6891Fb3c0b5281"
+CARTESI_CONTRACTS_AUTHORITY_ADDRESS="0x5050F233F2312B1636eb7CF6c7876D9cC6ac4785"
```

Comment:
I don't understand yet who is deploying this Authority (and History).

Reply:
sunodo run is still using `sunodo/devnet:1.1.1`, which is rather old and still have Authority and History contracts

Reply:
Should we deploy it on `sunodo/devnet` and bump it's version too?

Reply:
I think so. But we can do that in a different PR. This PR can work with the old version, which has the same smart contract code.